kythe/go/extractors/gcp/README.md

```diff
@@ -40,7 +40,7 @@ This also assumes you specify `_BUCKET_NAME` as per the Hello World Test above.
 ```
 gcloud builds submit --config examples/mvn.yaml \
 --substitutions=\
-_BUCKET_NAME="$BUCKET_NAME",\
```

Comment:
Why remove the quotes?

Reply:
they weren't necessary in the first place